src/AlgebraicGeometry/Schemes/PrincipalOpenSubset/Objects/Attributes.jl

```diff
@@ -18,7 +18,7 @@ complement_equation(U::PrincipalOpenSubset) = U.f::elem_type(OO(ambient_scheme(U
 ### assure compatibility with SpecOpen 
 gens(U::PrincipalOpenSubset) = [lifted_numerator(complement_equation(U))]
 ngens(U::PrincipalOpenSubset) = 1
-getindex(U::PrincipalOpenSubset, i::Int) = (i == 1 ? U : error("index out of range"))
+getindex(U::PrincipalOpenSubset, i::Int) = (i == 1 ? lifted_numerator(complement_equation(U)) : error("index out of range"))
```

Comment:
This method is actually called in at least one test (that's how I discovered the discrepancy), so at the very least that test would have to be updated as well.